### PR TITLE
Add Lagom monitors and graphs

### PR DESCRIPTION
**Pull request by jsravn**
Creates a critical alert if a circuit breaker trips. I set the threshold
very low on purpose since circuit breakers already have their own
thresholds, so if it trips something is really wrong.

Also adds graphs for the interesting circuit breaker metrics.

part of https://github.com/lightbend/es-backend/issues/339

### CODE REVIEW

File: enterprise-suite/es-monitor-api/bootstrap-monitors.json [hunk 1]
```diff
@@ -634,6 +634,21 @@
...
+    },
+    {
+      "model": "threshold",
+      "parameters": {
+        "name": "lagom_circuit_breaker_state",
+        "metric": "lagom_circuit_breaker_state",
+        "comparator": "<",
+        "threshold": "3",
+        "window": "1m",
+        "confidence": "0.0",
```
[the remark this answers is not in the document]
Comment (jsravn) [reply enterprise-suite/es-monitor-api/bootstrap-monitors.json]: @srikanthops why not?

**Comment by jsravn** [comment]
@srikanthops Not sure why confidence can't be 0 - but the frontend blows up anyways at 0. In fact, it seems to blow up with any small value - had to set it to 0.5 before it was happy. Seems like there is a bug lurking there.

**Comment by srikanthops** [comment]
I agree with you setting it close to zero, 
confidence from what I understand : health is already bad, what is your confidence to raise this alert.
confidence 0.3 window 2m => fire this alert if health is bad for 30% of time 2m window
confidence 0 =>  never fire this alert .. 
I could be wrong, but this is what I remember

**Comment by jsravn** [comment]
Build failing due to flake. https://github.com/lightbend/es-backend/issues/353

**Pull request merged**